content/docs/deployment/helm/modules/installation/authorization v2.0/_index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Authorization v2.0
+linktitle: "Authorization v2.0"
+description: >
+  Dell Technologies (Dell) Container Storage Modules (CSM) for Authorization v2.0 Helm deployment
+---
+
+{{% pageinfo color="primary" %}}
+The CSM Authorization karavictl CLI is no longer actively maintained or supported. It will be deprecated in a future release.
```

Comment:
Instead of publishing the deprecation which is already mentioned in 1.x pages, let's just indicate that karavictl will not work with Auth 2.x and users should use dellctl instead. This need not be a banner on top, it can be a "Note" in the 'Installing dellctl' section below.

Reply:
fixed